### PR TITLE
MODINVSTOR-473: Add indexes for *suppress properties

### PR DESCRIPTION
**Pull request by bohdan-suprun**
https://issues.folio.org/browse/MODINVSTOR-473 - Add b-tree indexes for staffSuppress and discoverySuppress properties.

### Changes:
* Add index for `instance.staffSuppress` and `instance.discoverySuppress` properties;
* Add index for `item.discoverySuppress` property;
* Add index for `holdingsRecord.discoverySuppress` property;

### CODE REVIEW

**Comment by marcjohnson-kint** [comment]
Are there already tests for searching with these properties?

File: src/main/resources/templates/db_scripts/schema.json [hunk 1]
```diff
@@ -608,6 +620,12 @@
           "tOps": "ADD",
           "caseSensitive": false,
           "removeAccents": false
+        },
+        {
```
Comment (marcjohnson-kint) [inline src/main/resources/templates/db_scripts/schema.json:624]: Does the `fromModuleVersion` need updating for indexes, or is there creation automatic irrespective of the `fromModuleVersion`?

Comment (bohdan-suprun) [reply src/main/resources/templates/db_scripts/schema.json:624]: I think @julianladisch mentioned that indexes will be created always regardless of `fromModuleVersion`.

**Comment by bohdan-suprun** [comment]
> Are there already tests for searching with these properties?

@marcjohnson-kint, I've added tests.

File: src/main/resources/templates/db_scripts/schema.json (hunk 1 above)
Comment (julianladisch) [reply src/main/resources/templates/db_scripts/schema.json:624]: Yes, we have automatic index recreation. For indexes `fromModuleVersion` is no longer needed.

**Pull request merged**